src/components/Editor/Editor.js

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+import './Editor.css';
+
+export default function Editor({
+  head,
+  setHead,
+  shirt,
+  setShirt,
+  bottom,
+  setBottom,
+  setCatchphrase,
+  setHeadCount,
+  setShirtCount,
+  setBottomCount,
+}) {
+  const [inputValue, setInputValue] = useState('');
+
+  const handleHead = (e) => {
+    setHead(e.target.value);
+    setHeadCount((currentState) => {
```

Comment:
generally wanna use ```prevState``` as you will see it all the time in other peoples' React apps